grails-app/views/common/_header.gsp

```diff
@@ -1,126 +1,2 @@
-<%@ page import="org.pih.warehouse.api.LocalizationApiController; org.pih.warehouse.core.RoleType" %>
-<!-- Block which includes the logo and login banner -->
-<g:if test="${grailsApplication.config.openboxes.helpscout.widget.enabled}">
-    <!--
-      -- Place magical HelpScout incantations early to make
-      -- window.Beacon accessible to any included divs, etc.
-      -->
-    <script type="text/javascript">
-      !function (e, t, n) {
-        function a() {
-          var e = t.getElementsByTagName("script")[0],
-            n = t.createElement("script");
-          n.type = "text/javascript", n.async = !0, n.src = "https://beacon-v2.helpscout.net", e.parentNode.insertBefore(n, e)
-        }
+<g:displayLogo location="${session?.warehouse?.id}" includeLink="${true}" />
```

Comment:
Why this is in a `_header` template if we do not have a header as such in the menu right now. Instead of rendering `_header` in the menu, you could now just use this tag lib there.